### PR TITLE
feat: [PROD-9516] Manage isDirty state for file and table parameters

### PR DESCRIPTION
**Pull request by ardvi**
Before creating a PR, please check that:
- [ ] Code is clean
- [ ] Tests are still working (cypress tests and `yarn test`)
- [ ] Changes don't cause new errors or warnings in browser console
- [ ] Documentation is up-to-date
- [ ] Breaking changes are clearly identified with [conventional commits](https://kapeli.com/cheat_sheets/Conventional_Commits.docset/Contents/Resources/Documents/index)

### CODE REVIEW

**Review by COSMO-EMI — APPROVED** [review]
LGTM :)

File: src/components/ScenarioParameters/ScenarioParameters.js [hunk 1]
```diff
@@ -224,6 +232,7 @@ const ScenarioParameters = ({
       launchScenario(scenarioId);
     }
     changeEditMode(false);
+    reset({ ...getValues() });
```
[the remark this answers is not in the document]
Comment (ardvi) [reply src/components/ScenarioParameters/ScenarioParameters.js:235]: Here we call the reset with the modified values, it's use here to update default values and mark parameters as not dirty.

File: src/components/ScenarioParameters/ScenarioParameters.js [hunk 2]
```diff
@@ -270,85 +279,87 @@ const ScenarioParameters = ({
...
-        <AccordionSummary
-          data-cy="scenario-params-accordion-summary"
-          className={classes.accordionSummary}
-          expandIcon={<ExpandMoreIcon />}
-          onClick={handleSummaryClick}
-        >
-          <Grid container className={classes.gridContainerSummary}>
-            <Grid className={classes.gridSummary}>
-              <Typography>{t('genericcomponent.text.scenario.parameters.title', 'Scenario parameters')}</Typography>
+      <ScenarioResetValuesContext.Provider value={scenarioResetValues}>
```
[the remark this answers is not in the document]
Comment (ardvi) [reply src/components/ScenarioParameters/ScenarioParameters.js:282]: I think it's better to put the provider higher as possible, it impact only component that use it.

**Review by csm-thu — APPROVED** [review]
LGTM :+1:

**Pull request merged**